### ISSUE TITLE
Cut window, impossible to resize.

### ISSUE THREAD

**Issue by boromyr** [sb53systems/G-Code-Flow-Temperature-Controller#12]
I think it's a DPI issue, but the 1.1 window is too big for the screen and not scalable, the bottom part is not accessible. I'm using a 125% DPI and a 1920x1080 resolution.
![image](https://github.com/user-attachments/assets/da4d6a53-8170-44b1-a6ec-7b15133f4ac4)

**Comment by sb53systems** [comment]
@boromyr yes, you need to use 100% DPI to display the interface correctly (I have the same 1920 1080 resol).  
![image](https://github.com/user-attachments/assets/276dff81-9504-4071-bdf8-a75940a88288)
![image](https://github.com/user-attachments/assets/15f7c4bd-8235-4432-b80e-cbd41147f69d)

**Comment by boromyr** [comment]
Isn't very convenient, do I have to change the dpi, log out and log back in every time to use the program?

**Comment by sb53systems** [comment]
Sorry about that, I'm working on this script alone. For the moment I've done the most important things, maybe later I'll add more.

**Comment by sb53systems** [comment]
Also be aware that this script is not supposed to be a definitive solution, it is an Exampl of use.  

Happy 3D Printing.

**Comment by sb53systems** [comment]
May add an option to hide or show the generated G-Code ?!

**Comment by boromyr** [comment]
Do you mean a wrapping of the two graphs with an arrow, to expand and reduce? Like this:
![image](https://github.com/user-attachments/assets/4743dbc2-aeed-407c-a40d-66ef878cdc82)
It could solve the problem, I reduce a graph to generate the gcode or to look at the other graph.

**Comment by sb53systems** [comment]
The Updated version of Sept 11 2024,  has the ScrollBar (Vert and Hori)  
![image](https://github.com/user-attachments/assets/fedb2e37-0bd6-4e58-b6cc-cd8caa6f34de)
![image](https://github.com/user-attachments/assets/20892705-bbc6-4b56-9c21-5ab86f703d0e)  
  
I think this will resolve your problem.  
  
Best regards.  
Salim.
